src/icrar/leap-accelerate-cli/main.cc

```diff
@@ -183,9 +191,10 @@ int main(int argc, char** argv)
     app.add_option("-s,--stations", rawArgs.stations, "Override number of stations to use in the measurement set");
     app.add_option("-f,--filepath", rawArgs.filePath, "MeasurementSet file path");
     app.add_option("-d,--directions", rawArgs.directions, "Direction calibrations");
-    app.add_option("-i,--implementation", rawArgs.implementation, "Compute implementation type (casa, eigen, cuda)");
+    app.add_option("-i,--implementation", rawArgs.implementation, "Compute implementation type (casa, cpu, cuda)");
     app.add_option("-c,--config", rawArgs.configFilePath, "Config filepath");
-    app.add_option("-m,--mwa-support", rawArgs.mwaSupport, "MWA data support by negating baselines");
+    //TODO: app.add_option("-m,--mwa-support", rawArgs.mwaSupport, "MWA data support by negating baselines");
```

Comment:
Still don't differentiate between mwa and ska baselines so I've disabled this argument for now, waiting to get richard's input about what this does